### PR TITLE
Add New CloudService Resource in Azure SDK

### PR DESCRIPTION
**Pull request by ashvermamsft**
We already have a merged PR https://github.com/Azure/azure-rest-api-specs/pull/11704 for swagger changes. We are starting this PR to merge the SDK changes.

itr1: 
1) Generated Model and Operations files from swagger (using public repo)
2) Added tests for clousServices resource, Fixed failing tests for Gallery due to latest swagger
3) Added session records and metadata files

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
please undo changes for non-cloudservice sdk updates

File: sdk/compute/Microsoft.Azure.Management.Compute/tests/Microsoft.Azure.Management.Compute.Tests.csproj [hunk 1]
```diff
@@ -26,6 +26,12 @@
     </None>
   </ItemGroup>
   
+  <ItemGroup>
+    <None Update="Resources\*.cspkg">
```
Comment (markcowl) [inline sdk/compute/Microsoft.Azure.Management.Compute/tests/Microsoft.Azure.Management.Compute.Tests.csproj:30]: This is likely the cause of your test failures - you should use '/' as the separator

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by ashvermamsft** [comment]
> please undo changes for non-cloudservice sdk updates

The files were generated using genSwagger.ps1, Do we want to segregate them? 
Also after generation there was build and test failure in gallery test. So I fixed them. Do let me know if we want to go ahead with gallery test failures.

File: sdk/compute/Microsoft.Azure.Management.Compute/tests/Microsoft.Azure.Management.Compute.Tests.csproj (hunk 1 above)
Comment (ashvermamsft) [reply sdk/compute/Microsoft.Azure.Management.Compute/tests/Microsoft.Azure.Management.Compute.Tests.csproj]: Have changed this, but no luck. This is unlikely to cause the tests to fail as SessionRecords are aslo being copied ( see above) The tests are passing locally, ran them in playback mode as well. Investigating why there is CI failures.

**Comment by grizzlytheodore** [comment]
@ashvermamsft yes please remove all non-cloudservices

**Comment by ashvermamsft** [comment]
> @ashvermamsft yes please remove all non-cloudservices

done.

**Comment by markcowl** [comment]
Holding, pending on discussion about the best way to handle CloudService APIs

**Comment by ashvermamsft** [comment]
Closing this PR and starting a new PR (https://github.com/Azure/azure-sdk-for-net/pull/17341)  after discussion with @markcowl and @grizzlytheodore

**Pull request closed without merging**